templates/javascript-ethers/scripts/whitelist-reader.js

```diff
@@ -0,0 +1,62 @@
+const fs = require("fs");
+const hre = require("hardhat");
+const { getServiceData, whitelistBeaconReader } = require("@api3/services");
+
+async function main() {
+  const network = hre.network.name;
+
+  // TODO: Update the mocked contract so that whitelisting is necessary
+  if (
+    network.toLowerCase() === "hardhat" ||
+    network.toLowerCase() === "localhost"
+  )
+    return;
+
+  // Read BeaconReaderExample contract address deployed using deploy.js
+  let beaconReaderExampleAddress = null;
+  try {
+    // TODO: Simplify
+    ({ beaconReaderExampleAddress } = JSON.parse(
+      fs.readFileSync(`./deployments/${network}.json`).toString()
+    ));
+    if (!beaconReaderExampleAddress) throw new Error("beaconId not found");
+  } catch (e) {
+    console.log(`Error: ${e}. Please try first running deploy script`);
+    return;
+  }
+
+  const deployments = getServiceData(
+    "Amberdata",
+    "global vwap for eth_usd",
+    network
+  );
+  beaconId = deployments.beacon.beaconId;
+
+  const account = hre.network.config.account;
```

Comment:
I wasn't aware you could get the account like that 🤔 

What I usually do is this instead:
```
const [signer, ...otherAccounts] = await ethers.getSigners();
```

Reply:
I am a big fan of ethers, but there are two things which I need to google everytime I use it:
1. how to parse logs
2. how to get my accounts in scripts

and you're thing is what I should use.